products/find-va-forms-redesign/form-detail-landing-pages/WIP-technical-specs-and-docs/va-forms-cms-governance.md

```diff
@@ -1,27 +1,31 @@
 # VA Forms - CMS Governance
 
-## Stages
 
-### Stage 1
-- The Forms DB is the source of truth for general form information (name, location, etc.). The CMS is updated by nightly migrations from the Forms DB.
-- The CMS is the source of truth for additional information (description, online tool, etc.) used on the form detail pages.
-
-### Stage 2
-- The Forms DB is retired
-- The CMS is the source of truth for all form data.
+## Stage 1
+- The VA Forms DB is the source of truth for general form information (form name, form number, form owner/ID, etc.). 
+- The CMS is updated by nightly migrations from the VA Forms DB.
+- The CMS is the source of truth for the additional forms information (related to, online tool, related forms/instructions, etc.) used on the form detail pages.
 
 ## Governance in Stage 1
-### Ownership 
-The VSA Public Websites team will own the all of the VA Forms content. Ownerwhip will be assigned during migration. Only CMS users with Public Websites team membership will be able to edit this content.
+
+### Ownership
+- The VSA Public Websites team will 'own' all of the VA forms content that originate in the CMS. Ownerwhip will be assigned during migration. Only Public Websites CMS users will be able to edit this content.
+- Forms data that originate in the VA Forms DB (e.g., form number, form owner/ID, form PDF URL) will continue to be owned by the forms managers and managed in the source VA Forms DB.
 
 ### Editorial workflow within CMS
 Form pages will be governed by the same workflow as other content ('Draft', 'In review' 'Published').
 
-### Permissions
+### CMS permissions
 
-- Because basic form data will be maintained in the Forms DB, non-admin users will not have permission to create or delete Form content in the CMS.
-- Users with appropriate roles and administration membership will have permission to edit Forms pages, but will not be able to edit fields that come from Forms DB.
+- Because basic form data will be maintained in the VA Forms DB, CMS users will not have the ability to create or delete form detail landing pages in the CMS. [@beth - not sure if I understood what this means, so I took a stab based on what I think I understand and what I would expect./jen] 
+- Users with appropriate roles and administration membership will have permission to edit form detail pages, but will not be able to edit fields that come from Forms DB. [Not sure about this. In stage 1, only PW content team with CMS access will have be able to edit or publish form detail pages; so "administration membership" doesn't apply, if I think this is what it means. For example: if a stakeholder or form manager tells us that a form should have different 'related to' labels, they'll have to get in touch with the PW team to have that updated for them./ jen]
```

Comment:
I think I used the wrong word here ('administration'). What would you call Public Websites in this context? Section, or team, maybe?

Reply:
PW content team. Like this...? 
"PW content team will have permission to edit form detail pages."  <--I'm assuming that any PW content person with editing permission can make edits to the editable fields; and that only the PW content people who have 'publish' permission can both make edits and publish updates. (Some PW content people will have edit access but not publish access.)

Reply:
PW content editor can edit form detail landing pages as 'draft,' but cannot publish. 

PW content publisher can edit, publish, and delete form detail landing pages.